cases/src/org/commcare/cases/instance/CaseInstanceTreeElement.java

```diff
@@ -107,7 +107,7 @@ public void attachStringCache(Interner<String> stringCache) {
      */
     public CaseChildElement getChild(String name, int multiplicity) {
         if (multiplicity == TreeReference.INDEX_TEMPLATE) {
-            return null;
+            return CaseChildElement.TemplateElement(this);
```

Comment:
Fix for case related queries failing in the wrong way when no case data has been loaded